test/functional/executor.py

```diff
@@ -62,8 +62,10 @@ def run(self):
                 print("TEST: ", test.get_name())
                 print("DESCRIPTION: ", test.get_description())
                 print("RUNNING: ", *cmd)
-            subprocess.run(cmd, stdout=subprocess.DEVNULL,
-                           stderr=subprocess.DEVNULL)
+                print(subprocess.getoutput(cmd))
```

Comment:
Thank you for this feedback. I will address those issues here. Let's have it sorted out.